cranelift/codegen/src/isa/x64/inst/args.rs

```diff
@@ -1238,6 +1244,20 @@ impl From<FloatCC> for FcmpImm {
     }
 }
 
+/// Encode the rounding modes used as part of the Rounding Control field.
+pub(crate) enum RoundImm {
+    RoundNearest = 0x00,
+    RoundDown = 0x01,
+    RoundUp = 0x02,
+    RoundZero = 0x03,
+}
+
+impl RoundImm {
+    pub(crate) fn encode(self) -> u8 {
+        self as u8
+    }
+}
```

Comment:
I noticed that `RoundImm` doesn't mention bit 2 (indicating where rounding is controlled) and bit 3 (the precision mask). Do you want to document this even if we don't implement that part? Right now we are implicitly setting both of those bits to 0.

Reply:
👍 Will add a note.